.rubocop-excludes.yml

```diff
@@ -0,0 +1,46 @@
+Lint/AmbiguousBlockAssociation:
```

Comment:
I'm a bit worried to have apps start with so many exceptions.. it feels like we are encouraging people to do it lightly. No need to do it right now, but maybe we can tackle it in the future. What do you think?

Reply:
I think we should have a deeper discussion about the cops and what do they enforce 🤓